File: Modul2/C/PRAK204-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>
#define phi 22/7

int main(){
    double r;
    double t;

    scanf("%lf %lf", &r, &t); printf("\n");

    double volume = (double) phi*r*r*t;
    double luas = (double) 2*phi*r*(r+t);
    double keliling = (double) 2*phi*r;

    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf", keliling);

    return 0;
}

File: Modul2/Python/PRAK204-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
import math

r, t = map(float, input().split())

phi = 22/7

volume = phi * r * r * t
luas = 2 * phi * r * (r + t)
keliling = 2 * phi * r

print(f"\nVolume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
